tests/Users/create-user.spec.js

```diff
@@ -0,0 +1,129 @@
+import { test, expect } from "@playwright/test";
+
+const getRandomNumber = () => {
+  return Math.floor(Math.random() * (999 - 100 + 1) + 100);
+};
+
+const wait = (seconds) => {
+  return new Promise((resolve) => {
+    setTimeout(resolve, seconds * 1000);
+  });
+};
+
+test.describe("Create user tests", () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto("./");
+    await page.getByPlaceholder("Email Address").fill("root@gmail.com");
+    await page.getByPlaceholder("Password").fill("p@s5W07d");
+    await page.getByRole("button", { name: "Log in" }).click();
+  });
+
+  test.afterEach(async ({ page }) => {
+    // await page.close();
+  });
+
+  test("Access Create User page and going back and forth", async ({ page }) => {
+    await page.getByRole('link', { name: 'Users' }).click();
+    await expect(page).toHaveURL('http://localhost:3000/users');
+    await expect(page.getByRole('heading', { name: 'Users table' })).toBeVisible();
+    await expect(page.getByRole('link', { name: 'Create user' })).toBeVisible();
+
+    await page.getByRole('link', { name: 'Create user' }).click();
+    await expect(page).toHaveURL('http://localhost:3000/users/create');
+    
+
+    await expect(page.getByRole('button', { name: 'Go Back' })).toBeVisible();
+    await wait(3)
+    await page.getByRole('button', { name: 'Go Back' }).click();
```

Comment:
Using wait(3) to pause test execution is generally discouraged because it introduces delays and can lead to flaky tests. We should be using Playwright's built-in wait functions, like waitForNavigation, waitForSelector, or waitForURL, to proceed when the application is ready.

Reply:
whats the best option here?

Reply:
await page.waitForTimeout(3000) is the only option here which is the same as our wait(3) function.